### ISSUE TITLE
Installing Vagrant's dokku-windows only shows nginx screen

### ISSUE THREAD

**Issue by mmahandev** [dokku/dokku#3598]
## Description of problem

Doing a fresh install of dokku-windows on vagrant shows a default "Welcome to nginx!" on [dokku.me](http://dokku.me/):

![image](https://user-images.githubusercontent.com/20646367/59974809-eeb40c00-9575-11e9-97cd-149da0a91504.png)

### Steps to Reproduce

Done by following http://dokku.viewdocs.io/dokku/getting-started/install/vagrant/

1. git clone https://github.com/dokku/dokku.git dokku-test
2. cd dokku-test
3. vagrant up dokku-windows
4. Setup SSH Config in ~/.ssh/config (Did this on the host machine)
5. Access [dokku.me](http://dokku.me/)

#### Actual Results

Shows an nginx page

#### Expected Results

Should show the dokku page

## Environment Information

Windows 10 64-bit. Vagrant 2.2.4

### `dokku report APP_NAME` output

<details>
<summary>I don't know what APP_NAME should be so I just ran `dokku report` after ssh'ing in:</summary>
-----> uname: Linux dokku 4.15.0-51-generic #55-Ubuntu SMP Wed May 15 14:27:21 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
-----> memory:
                     total        used        free      shared  buff/cache   available
       Mem:            985         128         194           3         662         700
       Swap:           979          23         956
-----> docker version:
       Client:
        Version:           18.09.6
        API version:       1.39
        Go version:        go1.10.8
        Git commit:        481bc77
        Built:             Sat May  4 02:35:57 2019
        OS/Arch:           linux/amd64
        Experimental:      false

       Server: Docker Engine - Community
        Engine:
         Version:          18.09.6
         API version:      1.39 (minimum version 1.12)
         Go version:       go1.10.8
         Git commit:       481bc77
         Built:            Sat May  4 01:59:36 2019
         OS/Arch:          linux/amd64
         Experimental:     false
-----> docker daemon info:
WARNING: No swap limit support
       Containers: 0
        Running: 0
        Paused: 0
        Stopped: 0
       Images: 2
       Server Version: 18.09.6
       Storage Driver: overlay2
        Backing Filesystem: extfs
        Supports d_type: true
        Native Overlay Diff: true
       Logging Driver: json-file
       Cgroup Driver: cgroupfs
       Plugins:
        Volume: local
        Network: bridge host macvlan null overlay
        Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
       Swarm: inactive
       Runtimes: runc
       Default Runtime: runc
       Init Binary: docker-init
       containerd version: bb71b10fd8f58240ca47fbb579b9d1028eea7c84
       runc version: 2b18fe1d885ee5083ef9f0838fee39b62d653e30
       init version: fec3683
       Security Options:
        apparmor
        seccomp
         Profile: default
       Kernel Version: 4.15.0-51-generic
       Operating System: Ubuntu 18.04.2 LTS
       OSType: linux
       Architecture: x86_64
       CPUs: 1
       Total Memory: 985.4MiB
       Name: dokku
       ID: WJIR:V6AK:CAUD:X5RE:TWOJ:I4AE:VPMI:YANC:DTHJ:MVBG:DTE2:OP6O
       Docker Root Dir: /var/lib/docker
       Debug Mode (client): true
       Debug Mode (server): false
       Registry: https://index.docker.io/v1/
       Labels:
       Experimental: false
       Insecure Registries:
        127.0.0.0/8
       Live Restore Enabled: false
       Product License: Community Engine

-----> sigil version: 0.4.0
-----> herokuish version:
       herokuish: 0.5.0
       buildpacks:
         heroku-buildpack-multi     v1.0.0
         heroku-buildpack-ruby      v200
         heroku-buildpack-nodejs    v142
         heroku-buildpack-clojure   v83
         heroku-buildpack-python    v149
         heroku-buildpack-java      v63
         heroku-buildpack-gradle    v28
         heroku-buildpack-scala     v84
         heroku-buildpack-play      v26
         heroku-buildpack-php       v151
         heroku-buildpack-go        v104
         buildpack-nginx            v10
-----> dokku version: dokku version v0.17.9-4-g23d647ee
-----> dokku plugins:
       plugn: 0.3.2
         00_dokku-standard    0.17.9 enabled    dokku core standard plugin
         20_events            0.17.9 enabled    dokku core events logging plugin
         app-json             0.17.9 enabled    dokku core app-json plugin
         apps                 0.17.9 enabled    dokku core apps plugin
         build-env            0.17.9 enabled    dokku core build-env plugin
         buildpacks           0.17.9 enabled    dokku core buildpacks plugin
         certs                0.17.9 enabled    dokku core certificate management plugin
         checks               0.17.9 enabled    dokku core checks plugin
         common               0.17.9 enabled    dokku core common plugin
         config               0.17.9 enabled    dokku core config plugin
         docker-options       0.17.9 enabled    dokku core docker-options plugin
         domains              0.17.9 enabled    dokku core domains plugin
         enter                0.17.9 enabled    dokku core enter plugin
         git                  0.17.9 enabled    dokku core git plugin
         logs                 0.17.9 enabled    dokku core logs plugin
         network              0.17.9 enabled    dokku core network plugin
         nginx-vhosts         0.17.9 enabled    dokku core nginx-vhosts plugin
         plugin               0.17.9 enabled    dokku core plugin plugin
         proxy                0.17.9 enabled    dokku core proxy plugin
         ps                   0.17.9 enabled    dokku core ps plugin
         repo                 0.17.9 enabled    dokku core repo plugin
         resource             0.17.9 enabled    dokku core resource plugin
         scheduler-docker-local 0.17.9 enabled    dokku core scheduler-docker-local plugin
         shell                0.17.9 enabled    dokku core shell plugin
         ssh-keys             0.17.9 enabled    dokku core ssh-keys plugin
         storage              0.17.9 enabled    dokku core storage plugin
         tags                 0.17.9 enabled    dokku core tags plugin
         tar                  0.17.9 enabled    dokku core tar plugin
         trace                0.17.9 enabled    dokku core trace plugin
</details>

### How (deb/make/rpm) and where (AWS, VirtualBox, physical, etc.) was Dokku installed?:

Installed via vagrant up dokku-windows.

### Additional information
N/A

**Comment by josegonzalez** [comment]
Can you create a new vm from scratch and show the output of the `vagrant up` command?

**Comment by MauriceMahan** [comment]
Hey @josegonzalez i'm using the dokku-windows so I ran that instead:

    $ vagrant up dokku-windows
    Bringing machine 'dokku-windows' up with 'virtualbox' provider...
    ==> dokku-windows: Checking if box 'bento/ubuntu-18.04' version '201906.18.0' is up to date...
    ==> dokku-windows: Machine already provisioned. Run `vagrant provision` or use the `--provision`
    ==> dokku-windows: flag to force provisioning. Provisioners marked to run always will still run.

For `vagrant ssh dokku-windows` I get this:
    $ vagrant ssh dokku-windows
    Welcome to Ubuntu 18.04.2 LTS (GNU/Linux 4.15.0-51-generic x86_64)

    * Documentation:  https://help.ubuntu.com
    * Management:     https://landscape.canonical.com
    * Support:        https://ubuntu.com/advantage

    System information as of Tue Jun 25 22:30:55 UTC 2019

    System load:  0.0               Users logged in:        0
    Usage of /:   6.6% of 61.80GB   IP address for eth0:    10.0.2.15
    Memory usage: 18%               IP address for eth1:    10.0.0.2
    Swap usage:   0%                IP address for docker0: 172.17.0.1
    Processes:    92


    20 packages can be updated.
    9 updates are security updates.

<details>
    <summary>If you really wanted the regular vagrant up code here it is:</summary>
    
    $ vagrant up
    Bringing machine 'dokku' up with 'virtualbox' provider...
    ==> dokku: Importing base box 'bento/ubuntu-18.04'...
    ==> dokku: Matching MAC address for NAT networking...
    ==> dokku: Checking if box 'bento/ubuntu-18.04' version '201906.18.0' is up to date...
    ==> dokku: Setting the name of the VM: dokku-dev_dokku_1561501744235_26322
    Vagrant cannot forward the specified ports on this VM, since they
    would collide with some other application that is already listening
    on these ports. The forwarded port to 8080 is already in use
    on the host machine.

    To fix this, modify your current project's Vagrantfile to use another
    port. Example, where '1234' would be replaced by a unique host port:

      config.vm.network :forwarded_port, guest: 80, host: 1234

    Sometimes, Vagrant will attempt to auto-correct this for you. In this
    case, Vagrant was unable to. This is usually because the guest machine
    is in a state which doesn't allow modifying port forwarding. You could
    try 'vagrant reload' (equivalent of running a halt followed by an up)
    so vagrant can attempt to auto-correct this upon booting. Be warned
    that any unsaved work might be lost.
</details>

**Comment by josegonzalez** [comment]
Is that "regular vagrant up" code from a scratch vm? I mean, after a `vagrant destroy`?

**Comment by MauriceMahan** [comment]
Oh I see. [Here's a pastebin](https://pastebin.com/8b8QMyH7) of the entire install after `vagrant up dokku-windows`.

**Comment by MauriceMahan** [comment]
I also got this when running `nginx -t` when ssh'd in

````
vagrant@dokku:~$ nginx -t
nginx: [alert] could not open error log file: open() "/var/log/nginx/error.log" failed (13: Permission denied)
2019/07/06 14:27:48 [warn] 20966#20966: the "user" directive makes sense only if the master process runs with super-user privileges, ignored in /etc/nginx/nginx.conf:1
nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
2019/07/06 14:27:48 [emerg] 20966#20966: open() "/run/nginx.pid" failed (13: Permission denied)
nginx: configuration file /etc/nginx/nginx.conf test failed
````

**Comment by josegonzalez** [comment]
Can anyone else on a Windows laptop replicate this? Regardless, I don't have a Windows machine to test this on, so I'll need to somehow acquire one to do so.

I might be able to install it via bootcamp on an older MacBook. @MauriceMahan are you using Windows 10 Home, Professional, or some other version? Also, are you using Virtualbox, VMWare, Parallels, or something else for the VM?

**Comment by MauriceMahan** [comment]
I'm using Windows 10 Professional.

**Comment by akbo** [comment]
I can't try this on windows as I have no windows machine but when I ran vagrant up on my Mac, it showed the config website on port 2000 instead of port 80. @mmahandev can you try if this is the case for you, too?

**Comment by pickettd** [comment]
I am just trying Dokku for the first time today and hit this issue. I'm using Windows 10 Professional and have Vagrant/Virtualbox working with other projects.

I noticed in the Vagrantfile that for the dokku-windows box it does have not a shell provisioner that calls `make dokku-installer` (like the regular dokku box has). So in my VM I tried `sudo make dokku-installer` from the dokku root directory. Now when I try to navigate to http://dokku.me I get a 502 Bad Gateway from nginx.

Also @akbo's comment says to try a different port. It looks like from the Vagrantfile that 8080 is supposed to be the forwarded port to host. When I try to go to http://dokku.me:8080 I get an error of site can't be reached and when I go to http://localhost:8080 I get the same 502 Bad Gateway as above (now that I have run `make dokku-installer` of course)

**Comment by davidpodhola** [comment]
Thanks @pickettd, just confirming this was an issue for me too. Running the dokku web-ui installer installer manually with `make dokku-installer` from `/root/dokku` started the installer.

**Comment by josegonzalez** [comment]
Can someone attach the output of:

```
vagrant up dokku-windows
```

from a fresh local run? As in, not from an existing `vagrant up`.

**Comment by davidpodhola** [comment]
sure have a look here please https://gist.github.com/davidpodhola/4c32c9c378ee1907d4b28507a940724f

**Comment by josegonzalez** [comment]
Closed (for now) by #3757. Someday I'll get a windows laptop and will actually figure out whats going on.
